package.json

```diff
@@ -11,11 +12,11 @@
   "repository": "https://github.com/AtomLinter/linter-markdown",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": "^1.0.0"
```

Comment:
atom does not validate the semver range, just that this prop exists - https://github.com/atom/atom/search?utf8=%E2%9C%93&q=engines

npm (which apm uses) uses the `semver` package to check the range - https://docs.npmjs.com/files/package.json#engines